src/components/Icon.tsx

```diff
@@ -62,11 +66,15 @@ const ICONS = {
   'hand-no-province-yet': HandNoProvinceYet,
   'canada-logo': CanadaLogo,
   'hand-reminder-red': HandReminderRed,
-  'qr-code': QRCode,
   'scan-qr-code-white-arrow': ScanQRCodeWhiteArrow,
   'camera-permission': Camera,
   'green-circle-check': GreenCircleCheck,
   'red-circle-exclamation': RedCircleExclamation,
+  'qr-code-icon': QRCodeIcon,
+  'hamburger-menu': HamburgerMenu,
+  close: Close,
```

Comment:
Missing quotes here?

Reply:
My linter told me not to. They are just keys in a javascript object so they don't need them, unless they are [not a valid js identifier](https://stackoverflow.com/questions/4348478/what-is-the-difference-between-object-keys-with-quotes-and-without-quotes), which is the case if there is a dash in the name.